Lavalink/Dockerfile

```
@@ -4,14 +4,18 @@ FROM eclipse-temurin:17-jre-alpine
 WORKDIR /app
 
 # Install required system libraries and utilities
+# eudev-libs: satisfies OSHI/LinuxOperatingSystem’s libudev probe (harmless if missing for Lavalink itself).
 RUN apk add --no-cache \
```

Hadolint (2.14.0)

[warning] Pin versions in apk add. Instead of `apk add <package>` use `apk add <package>=<version>`

(DL3018)